interface/src/app/gene/network/network.js

```diff
@@ -235,16 +255,29 @@ angular.module('adage.gene.network', [
           .call(edgeTip);
 
         geneTip.html(getGeneInfo);
-        network.onGene('mouseover.custom', geneTip.show);
-        network.onGene('mouseout.custom', geneTip.hide);
-        network.onEdge('mouseover.custom', showEdgeTip);
-        network.onEdge('mouseout.custom', hideEdgeTip);
+        network.onGene('click.custom', geneTip.show);
+        network.onEdge('click.custom', showEdgeTip);
 
         // Draw network svg with legend and filter.
         network.showLegend()
           .filter(0, self.maxNodeNum)
           .draw();
-      }
+
+        // Add event handlers to gene-tip and edge-tip so that they will be
+        // hidden when clicked:
+        var addClickHandler = function(className, handler) {
+          var elements = document.getElementsByClassName(className);
+          var i, n;
+          if (elements) {
+            n = elements.length;
+            for (i = 0; i < n; ++i) {
+              elements[i].onclick = handler;
+            }
+          }
+        };
+        addClickHandler('gene-tip', geneTip.hide);
+        addClickHandler('edge-tip', hideEdgeTip);
```

Comment:
That's true. Since the edge tip now can be hidden only by clicking the the tip window, it can be hidden synchronously. I will remove this "workaround" function.